src/edu/psu/sweng500/team8/gui/SudokuGUI.java

```diff
@@ -66,10 +80,22 @@ public void cellChanged(Cell cell, int newNumber) {
 		clearMessage();
 		this.gameBoard.clearHighlightedIncorrectCells();
 		
+		this.board = this.gameSession.getGameBoard();
```

Comment:
This shouldn't be necessary. I don't have a problem with adding it, but then we should use this consistently throughout.